src/main/kotlin/DataClasses.kt

```diff
@@ -0,0 +1,16 @@
+//Дата-класс Заметка
+data class Note(val name: String, var body: String = "") {
+    override fun toString(): String {
+        return ("Имя заметки: $name \nТекст заметки: $body")
+    }
+}
+
+//Дата-класс Архив - список заметок
+data class Archive(val name: String) {
+    var body = mutableListOf<Note>()
```

Comment:
🍏 По умолчанию в Kotlin поля, переменные являются публичными. Это значит, что сейчас легко переопределить значение body извне. Рекомендую  сделать body неизменяемым полем. При этом внутри у него будет изменяемый список, который можно редактировать:
```suggestion
    val body = mutableListOf<Note>()
```